3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,131 @@
+
+let yearOfBirth = 1982;
```

Comment:
it is better to use const if the value does not change.

could you please check all variables and change it to const if the value does not change

Reply:
Hello! I have noticed that I have made a mistake with the name of my branch, and I find two files that are not mine, one is this script.js and another is index.html in the week1 folder. I separated my files into individual folders per task. Should I create a new branch with my name so as not to overwrite anything? Nico Q